v3.0, dev/tidb-in-kubernetes/faq: update FAQ (#1382)
pingcap/docs: dev/tidb-in-kubernetes/faq.md

    * In the `values.yaml` file of the TiDB cluster, modify `timezone` settings in the `values.yaml` file of the TiDB cluster. For example, you can set it to `timezone: Asia/Shanghai` and then upgrade the TiDB cluster.
    * Refer to [Time Zone Support](/how-to/configure/time-zone.md) to modify TiDB service time zone settings.

## Can HPA or VPA be configured on TiDB components?

> Currently, TiDB cluster does not support HPA (Horizontal Pod Autoscaling) or VPA (Vertical Pod Autoscaling), because it is difficult to achieve autoscaling on applications with state such as database. Autoscaling can not be achieved merely by the monitoring data of CPU and memory.

Reviewer: ```suggestion
Currently, the TiDB cluster does not support HPA (Horizontal Pod Autoscaling) or VPA (Vertical Pod Autoscaling), because it is difficult to achieve autoscaling on stateful applications such as a database. Autoscaling can not be achieved merely through the monitoring data of CPU and memory.
```

> ## What scenarios require manual intervention when I use TiDB Operator to orchestrate TiDB clusters?

Reviewer: ```suggestion
## What scenarios require manual intervention when I use TiDB Operator to orchestrate a TiDB cluster?
```

> Except for the operation of the Kubernetes cluster itself, TiDB Operator has the following two scenarios that might require manual intervention:

Reviewer: ```suggestion
Besides the operation of the Kubernetes cluster itself, there are the following two scenarios that might require manual intervention when using TiDB Operator:
```

> * Adjusting the cluster after the auto-failover of TiKV. Detailed reference: [Auto-Failover](tidb-in-kubernetes/maintain/auto-failover.md)

Reviewer: ```suggestion
* Adjusting the cluster after the auto-failover of TiKV. See [Auto-Failover](tidb-in-kubernetes/maintain/auto-failover.md) for details;
```

> * Maintaining or dropping the specific Kubernetes nodes. Detailed reference: [Maintaining Nodes](tidb-in-kubernetes/maintain/kubernetes-node.md)

Reviewer: ```suggestion
* Maintaining or dropping the specified Kubernetes nodes. See [Maintaining Nodes](tidb-in-kubernetes/maintain/kubernetes-node.md) for details.
```

## What is the recommended deployment topology when I use TiDB Operator to orchestrate a TiDB cluster on a public cloud?

> To achieve high availability and data security, it is recommended for you to deploy at least three Available Zones in a TiDB cluster in a production environment.

Reviewer: ```suggestion
To achieve high availability and data security, it is recommended that you deploy the TiDB cluster in at least three availability zones in a production environment.
```
See [Running in multiple zones](https://kubernetes.io/docs/setup/best-practices/multiple-zones/)

> For the relationship of deployment topology between TiDB cluster and TiDB services, TiDB Operator supports the following three deployment modes, each of which has its own merits and demerits. Specific selection must be based on actual business needs.

Reviewer: ```suggestion
In terms of the deployment topology relationship between the TiDB cluster and TiDB services, TiDB Operator supports the following three deployment modes. Each mode has its own merits and demerits, so your choice must be based on actual application needs.
```

> * Deploy TiDB clusters and TiDB services in the same Kubernetes cluster in the same VPC;

Reviewer: ```suggestion
* Deploy the TiDB cluster and TiDB services in the same Kubernetes cluster of the same VPC;
```

> * Deploy TiDB clusters and TiDB services in different Kubernetes clusters in the same VPC;

Reviewer: ```suggestion
* Deploy the TiDB cluster and TiDB services in different Kubernetes clusters of the same VPC;
```

> * Deploy TiDB clusters and TiDB services in different Kubernetes clusters in different VPCs.

Reviewer: ```suggestion
* Deploy the TiDB cluster and TiDB services in different Kubernetes clusters of different VPCs.
```

> ## Does TiDB Operator supports TiSpark?

Reviewer: ```suggestion
## Does TiDB Operator support TiSpark?
```

> TiS Operator does not yet support the automatic orchestration of TiSpark.

Reviewer: ```suggestion
TiDB Operator does not yet support automatically orchestrating TiSpark.
```

> If you want to add TiSpark components to TiDB in Kubernetes, you must maintain Spark on your own in **the same** Kubernetes cluster. You must ensure that Spark can access the IP and ports of PD and TiKV instances, and install the TiSpark plugin for Spark. [TiSpark](/reference/tispark.md#deploy-tiSpark-on-the-existing-spark-cluster) offers a detailed guide for you to install the TiSpark plugin.

Reviewer: ```suggestion
If you want to add the TiSpark component to TiDB in Kubernetes, you must maintain Spark on your own in **the same** Kubernetes cluster. You must ensure that Spark can access the IPs and ports of PD and TiKV instances, and install the TiSpark plugin for Spark. [TiSpark](/reference/tispark.md#deploy-tiSpark-on-the-existing-spark-cluster) offers a detailed guide for you to install the TiSpark plugin.
```

> Refer to [Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html) to maintain Spark in Kubernetes.

Reviewer: ```suggestion
To maintain Spark in Kubernetes, refer to [Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html).
```

> ## How to check the configuration of TiDB cluster?

Reviewer: ```suggestion
## How to check the configuration of the TiDB cluster?
```
